homeassistant/components/websocket_api/commands.py

```diff
@@ -151,9 +151,11 @@ def handle_unsubscribe_events(hass, connection, msg):
         connection.send_message(messages.error_message(
```

Comment:
Why not log `ServiceNotFound` too?

Reply:
There is a unit test specific testing we don't log error for service not found. I bet there was some story about it.

Reply:
We don't log it because we are actually already reporting it back to the frontend: `ERR_NOT_FOUND`. The other two errors are more generic.